src/webgpu/api/validation/resource_usages/textureUsageInRender.spec.ts

```diff
@@ -115,42 +230,65 @@ g.test('readwrite_upon_subresources')
     pass.setBindGroup(0, bindGroup);
     pass.endPass();
 
+    const success = _resourceSuccess || _usageSuccess;
     t.expectValidationError(() => {
       encoder.finish();
-    }, !_success);
+    }, !success);
   });
 
-g.test('readwrite_upon_aspects')
+g.test('subresources_and_binding_types_combination_for_aspect')
   .params(
     params()
       .combine(poptions('format', ['depth32float', 'depth24plus', 'depth24plus-stencil8'] as const))
```

Comment:
Maybe this shouldn't be a hardcoded list of formats?

Reply:
I didn't see a small and simple change for this. Maybe we can split all texture formats (kTextureFormatInfo) into color formats (kColorTextureFormatInfo) and depth/stencil formats (kDepthStencilTextureFormatInfo) in capability_info.ts? @Kangz and @kainino0x , WDYT?

Reply:
I've been wanting to do this for a while and with compressed formats it became more pressing. Now being done in #263!

Reply:
Thanks!